.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = tab
+indent_size = 1
```

Comment:
Is this configuration perhaps the reason that github sometimes renders the tabs as 1 space? https://stackoverflow.com/questions/8833953/how-to-change-tab-size-on-github/23522945#23522945 
Perhaps it would be better to be agnostic as to how they render in a particular interface and leave this line out, allowing the rendering to default to whatever the user has locally - or else adopt a "middle of the road" default such as 4

Reply:
Very good point! The `indent_size` config is removed. Now the Github review UI renders 4 spaces for a tab indentation.